composer.json

```diff
@@ -16,18 +16,18 @@
         }
     },
     "require": {
-        "php": ">=5.3.0",
+        "php": "^7.2",
         "ext-soap": "*",
-        "symfony/options-resolver": "~2.6|~3.0",
+        "symfony/options-resolver": "^3.4.37",
         "symfony/polyfill-iconv": "^1.2"
     },
     "require-dev": {
         "kasperg/phing-github": "0.2.*",
-        "phing/phing": "~2.7",
+        "phing/phing": "^2.7",
         "php-vcr/php-vcr": "1.2.6 - 1.2.7|^1.3.1",
-        "php-vcr/phpunit-testlistener-vcr": "~1.1.2",
-        "phpunit/phpunit": "~4.4",
-        "psr/log": "~1.0",
-        "symfony/yaml": "~2.1"
+        "php-vcr/phpunit-testlistener-vcr": "^1.1.2",
+        "phpunit/phpunit": "^4.8.36",
```

Comment:
That version of PHPUnit is officially not compatible with php 7.2.

Reply:
I know, see #387. I'm currently just trying to fix this Travis CI check.

Reply:
I had to upgrade to at least PHPUnit 6